lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,542 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli
+short_description: Manages Kubernetes packages with the Helm package manager
+description:
+  - Install, upgrade, delete packages with the Helm package manager.
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+notes:
+  - Support for helm3 is not finished waiting fix on helm 3
```

Comment:
I'm not sure what this is supposed to mean. "Waiting fix on helm3"? Which fix?

Reply:
Since stable Helm v3 is coming soon (it hit rc 3 recently and new pre-releases are being published frequently) is would be good to support also Helm 3 out-of-box.

Reply:
I'm working on it !
I have another branch where the module works with last release of helm3
I haven't merge it yet because I add some integration tests on both helm2 and helm3

Reply:
Helm 3 support is now ready and tested